test/train-sets/ref/help.stdout

```diff
@@ -214,6 +214,16 @@ Experience Replay:
                               example be played (default: 1 = permuting)
 Offset tree Options:
   --ot arg              Offset tree with <k> labels
+Contextual Bandit Options:
```

Comment:
not happy with --help output

Reply:
nit, suggestions: 
old->legacy
cbexploreadf -> cb_explore_adf

Maps cb_adf to cb -> shouldn't that be the other way around, i.e. cb to cb_adf?